query-node/substrate-query-framework/cli/src/templates/entities/model.ts.mst

```diff
@@ -17,6 +17,8 @@ import {
   StringField
 } from 'warthog';  {{! we don't need extra twists here }}
 
+{{#has.numeric}}import * as BN from 'bn.js' {{/has.numeric}}
```

Comment:
Do we need to add `bn.js` as a dependency to the  `generated/graphql-server/package.json` ?

Reply:
Yes we do, for numeric columns `BN` is the typescript type.